### PR TITLE
[SPARK-26324][DOCS] Add Spark docs for Running in Mesos with SSL

### PR DESCRIPTION
**Pull request by jomach**
## What changes were proposed in this pull request?
Added docs for running spark jobs with Mesos on SSL

### CODE REVIEW

File: docs/running-on-mesos.md [hunk 1]
```diff
@@ -108,6 +108,19 @@ Please note that if you specify multiple ways to obtain the credentials then the
 
 An equivalent order applies for the secret.  Essentially we prefer the configuration to be specified directly rather than indirectly by files, and we prefer that configuration settings are used over environment variables.
 
+### Deploy to a Mesos running on Secure Sockets
+
+If you want to deploy a Spark Application into a Mesos cluster that is running in a secure mode there are some environment's variables that need to be set.
```
Comment (felixcheung) [inline docs/running-on-mesos.md]: `environment variables`

File: docs/running-on-mesos.md [hunk 2]
```diff
@@ -108,6 +108,19 @@ Please note that if you specify multiple ways to obtain the credentials then the
...
+
+- `LIBPROCESS_SSL_ENABLED=true` enables SSL communication
+- `LIBPROCESS_SSL_VERIFY_CERT=false` verifies the ssl certificate 
+- `LIBPROCESS_SSL_KEY_FILE=pathToKeyFile.key` path to key 
+- `LIBPROCESS_SSL_CERT_FILE=pathToCRTFile.crt` the certificate file to be used
+
+all options can be found at http://mesos.apache.org/documentation/latest/ssl/
+
+Then submit as usual or as described below
```
Comment (felixcheung) [inline docs/running-on-mesos.md]: perhaps say `as described in Client mode or Cluster mode below`

File: docs/running-on-mesos.md [hunk 3]
```diff
@@ -108,6 +108,19 @@ Please note that if you specify multiple ways to obtain the credentials then the
...
+### Deploy to a Mesos running on Secure Sockets
+
+If you want to deploy a Spark Application into a Mesos cluster that is running in a secure mode there are some environment's variables that need to be set.
+
+- `LIBPROCESS_SSL_ENABLED=true` enables SSL communication
+- `LIBPROCESS_SSL_VERIFY_CERT=false` verifies the ssl certificate 
+- `LIBPROCESS_SSL_KEY_FILE=pathToKeyFile.key` path to key 
+- `LIBPROCESS_SSL_CERT_FILE=pathToCRTFile.crt` the certificate file to be used
+
+all options can be found at http://mesos.apache.org/documentation/latest/ssl/
```
Comment (srowen) [inline docs/running-on-mesos.md]: all -> All

**Review by srowen — CHANGES_REQUESTED** [review]

**Comment by jomach** [comment]
@srowen @felixcheung  pr updated

**Review by srowen — APPROVED** [review]

**Comment by jomach** [comment]
@srowen Why did you not merge it ?

**Comment by srowen** [comment]
It is merged to master. I missed running the tests so those are running now, but it's a doc change and shouldn't be a problem

**Comment by jomach** [comment]
I saw it. I was just confused about the git symbol. Thx

**Comment by srowen** [comment]
That error can be ignored; this didn't cause functional changes in Python

**Pull request closed without merging**